lib/api/lead.js

```diff
@@ -57,6 +57,13 @@ Lead.prototype = {
       });
   },
 
+  associateLead: function(leadId, cookieId) {
+    var path = util.createPath('leads', leadId, 'associate.json');
+    path = util.appendParam(path, "cookie", cookieId);
+
+    return this._connection.postJson(path);
```

Comment:
The request code is based on [restler](https://github.com/danwrong/restler). This means you can also pass query data with `postJson` without having to do it manually:

```js
this._connection.postJson(path, {}, {query: {cookie: cookieId}})
```

I'll merge the code and make this change. This means that I'll also remove `appendParam` since that's also obsolete.